.node-version

```diff
@@ -0,0 +1 @@
+18.16.0
```

Comment:
this should be 18.7.0 as is the one used in the GHA or anything between 18 and 19

Reply:
We can also change the GHA to use this file by changing
```yml
with: 
  version: "${{ env.NODE_VERSION }}"
```
to
```yml
with:
  node-version-file: ".node-version"
```
like we have in the web app

Reply:
awesome i will open a follow up PR to reference that